index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    />
+    <link rel="stylesheet" href="/css/global.css" />
+    <link rel="stylesheet" href="/css/style.css" />
+  </head>
+  <body>
+    <div id="wrap">
+      <header class="header-container">
+        <a class="logo-link" href="/">
+          <img src="/images/logo@2x.png" alt="logo" width="153" height="51" />
+        </a>
+        <a class="login-link" href="/login">로그인</a>
+      </header>
+      <main class="main-container">
+        <div class="mt-70"></div>
+        <article class="banner">
+          <div class="banner-container">
+            <div class="banner-info">
+              <h2>
+                일상의 모든 물건을<br />
+                거래해 보세요
+              </h2>
+              <a class="banner-link" href="/items">구경하러 가기</a>
+            </div>
+            <img src="images/Img_home_top@3x.png" width="996" height="447" />
+          </div>
+        </article>
+        <section class="container">
+          <div class="content">
+            <img src="images/Img_home_01@3x.png" />
+            <div class="content-text">
+              <h6>Hot item</h6>
```

Comment:
```suggestion
              <span>Hot item</span>
```

`h1` ~ `h6`을 사용할때는 의미단위로 사용해야합니다 예를들어 `h6`을 사용하려면 이것은 중요도가 가장 낮은 제목이란 뜻인데 그 전에 `h5`가 사용이 되어야합니다.
`span` 태그를 사용하고 css로 크기를 조절해주세요.